ios/app/PrivacyInfo.xcprivacy

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We need to put this into the SDK instead.

Reply:
Moved.